app/pages/about/team-page.cjsx

```diff
@@ -59,11 +59,6 @@ counterpart.registerTranslations 'en',
         bio: '''Coleman is helping to create new data analysis and visualization tools for existing Zooniverse
         projects as well as identifying new projects within the University of Portsmouth. He received
         his PhD in astrophysics from Drexel University.'''
-      darrenMcRoy:
```

Comment:
Normally best to split out concerns when issuing PR's, keeps the logic of review nice and succinct.